src/main_app/jobs_workers/public_jobs_workers/copy_svg_langs_per_file/objects.py

```
@@ -0,0 +1,87 @@
+        Converts the dataclass instance back to its original dictionary format.
+        """
+
+        return asdict(self)
+
+
+__all__ = [
+    "StepResult",
+    "FileSteps",
+    "FilesProcessedItem",
+    "CopySvgLangsPerFileWorkerObject",
+]
```

Ruff (0.15.17)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)